stdlib/csv/csv_test.flux

```diff
@@ -0,0 +1,54 @@
+package csv_test
+
+
+import "array"
+import "testing"
+import "csv"
+
+testcase from_raw{
+    input = "
+time,float,int,uint,bool,string
+2021-03-12T13:58:59Z,42.69,-67,152,false,hello world
+2021-03-12T13:58:59Z,42.69,-67,152,false,hello world
+2021-03-12T13:58:59Z,42.69,-67,152,false,hello world
+2021-03-12T13:58:59Z,42.69,-67,152,false,hello world
+"
+    want = array.from(
+        rows: [
+            {time: "2021-03-12T13:58:59Z", float: "42.69", int: "-67", uint: "152", bool: "false", string: "hello world"},
+            {time: "2021-03-12T13:58:59Z", float: "42.69", int: "-67", uint: "152", bool: "false", string: "hello world"},
+            {time: "2021-03-12T13:58:59Z", float: "42.69", int: "-67", uint: "152", bool: "false", string: "hello world"},
+            {time: "2021-03-12T13:58:59Z", float: "42.69", int: "-67", uint: "152", bool: "false", string: "hello world"},
+        ],
+    )
+
+    // Using raw mode so all columns are of type string
+    result = csv.from(csv: input, mode: "raw")
+
+    testing.diff(got: result, want: want)
+}
+testcase from_annotations{
+    input = "
+#datatype,string,long,dateTime:RFC3339,double,long,unsignedLong,boolean,string
+#group,false,false,false,false,false,false,false,false
+#default,_result,,,,,,,
+,result,table,time,float,int,uint,bool,string
+,,0,2021-03-12T13:58:59Z,42.69,-67,152,false,hello world
+,,0,2021-03-12T13:58:59Z,42.69,-67,152,false,hello world
+,,0,2021-03-12T13:58:59Z,42.69,-67,152,false,hello world
+,,0,2021-03-12T13:58:59Z,42.69,-67,152,false,hello world
+"
+    want = array.from(
+        rows: [
+            {time: 2021-03-12T13:58:59Z, float: 42.69, int: -67, uint: uint(v: 152), bool: false, string: "hello world"},
+            {time: 2021-03-12T13:58:59Z, float: 42.69, int: -67, uint: uint(v: 152), bool: false, string: "hello world"},
+            {time: 2021-03-12T13:58:59Z, float: 42.69, int: -67, uint: uint(v: 152), bool: false, string: "hello world"},
+            {time: 2021-03-12T13:58:59Z, float: 42.69, int: -67, uint: uint(v: 152), bool: false, string: "hello world"},
+        ],
+    )
+
+    // Using annotations mode so all columns can be specific types
+    result = csv.from(csv: input, mode: "annotations")
+
+    testing.diff(got: result, want: want)
+}
```

Comment:
@rockstar  Could you review this test case to make sure I did it right?

Reply:
Also are these tests part of Flux CI? We created an issue for influxdb and IDPE but did we do it for Flux?

Reply:
I am really liking the new test format.